cli/tree/src/main/scala/edu/rpi/tw/twxplore/cli/tree/command/EtlCommand.scala

```diff
@@ -15,8 +23,230 @@ object EtlCommand extends Command {
 
   val args = new Args()
 
+  def replaceComma(str: String, startIndex: Int, endIndex: Int): String = {
+    val result: String = str.substring(0, startIndex) + str.substring(startIndex+1, endIndex).replace(",", "+") + str.substring(endIndex+1)
+    result
+  }
+
   def apply(): Unit = {
-//    val pipelineName = args.pipelineName.toLowerCase()
+    val pipelineName = args.pipelineName.toLowerCase()
+    val dataDirectoryPath = Paths.get(args.dataDirectoryPath)
+    val source = scala.io.Source.fromFile(args.dataDirectoryPath)
+    var treeList: ListBuffer[Tree] = new ListBuffer[Tree]()
+    var treeSpeciesList: mutable.HashMap[String, TreeSpecies] = new mutable.HashMap()
+    var boroughList: mutable.HashMap[Int, Borough] = new mutable.HashMap()
+    var ntaList: mutable.HashMap[String, NTA] = new mutable.HashMap()
+    var blockList: mutable.HashMap[Int, Block] = new mutable.HashMap()
+    var postalCode: mutable.HashMap[Int, Postcode] = new mutable.HashMap()
+    val state: State = State("New York", new ListBuffer[City])
+    val city: City = City("New York City", new ListBuffer[Borough], new ListBuffer[Postcode], state.state)
+    state.cities += city
+    var line_no = 0
+
+    for(line <- source.getLines) {
+      line_no += 1
+      breakable {
```

Comment:
Using continue/break in Scala is really poor form. Decompose into functions. You could also have a nested LineProcessor class which has all of your state (maps, etc.), then process(line: String).